query/internal/gota/cmo_test.go

```diff
@@ -15,7 +15,7 @@ func TestCMO(t *testing.T) {
 		}
 	}
 
-	if diff := diffFloats(expList, actList, 1E-7); diff != "" {
+	if diff := diffFloats(expList, actList, 1e-7); diff != "" {
```

Comment:
were these automatically changed by the formatter?

Reply:
yes